graphql/query_resolver.go

```diff
@@ -443,7 +443,7 @@ func (r *queryResolver) Projects(ctx context.Context) ([]*GroupedProjects, error
 	if err != nil {
 		return nil, InternalServerError.Send(ctx, fmt.Sprintf("error getting viewable projects for '%s': '%s'", usr.DispName, err.Error()))
 	}
-	allProjects, err := model.FindMergedProjectRefsByIds(viewableProjectIds...)
+	allProjects, err := model.FindMergedTrackedProjectRefsByIds(viewableProjectIds...)
```

Comment:
I think this is the wrong change. groupProjects already filters out hidden projects. Looking at https://github.com/evergreen-ci/evergreen/pull/7627/files it looks like what's missing is filtering out disabled projects.

Can you add a unit test to this? I think that would've increased confidence that your change actually did anything (if you added a test for hidden, it would've succeeded with or without this change, which would've told you this wasn't the right change).

Reply:
🤦 good catch! I didn't realize there was a difference between enabled and hidden